docs/Users_Guide/matthewjetstream.rst

```diff
@@ -192,68 +211,101 @@ When the installation is complete, you can verify that the Docker command line t
 
     docker --version
 
-Next, you must start the Docker daemon, which runs in the background and processes commands::
+The Docker daemon should start automatically, but it sometimes runs into issues.
+First, check to see if the daemon started successfully::
 
-    sudo service docker start
+    sudo systemctl --no-pager status docker
 
-If that command appeared to succeed, you can confirm its status with this command::
+If you see a message saying the daemon failed to start because a "Start request repeated too quickly",
+wait a few minutes and issue this command to try again to start it::
 
-    sudo systemctl --no-pager status docker
+    sudo systemctl start docker
+
+If the command seems to succeed, confirm that the daemon is running using the status command above.
+Repeat these efforts as necessary until it is started.
 
-Once all of that is in order, you must pull the latest version of the I-WRF image onto your instance::
+Get the WRF and METplus Docker Images and the Observed Weather Data
+-------------------------------------------------------------------
 
-    docker pull ncar/iwrf
+Once Docker is running, you must pull the correct versions of the WRF and METplus images onto your instance::
+
+    docker pull ${WRF_IMAGE}
+    docker pull ${METPLUS_IMAGE}
+
+The METplus analysis will be comparing the results of the WRF simulation against
+the actual weather data that was recorded during Hurricane Matthew.
+We download that data by pulling a Docker volume that holds it,
+and then referencing that volume when we run the METplus Docker container.
+The commands to pull and create the volume are::
+
+    docker pull ncar/iwrf:data-matthew-input-obs.docker
+    docker create --name ${OBS_DATA_VOL} ncar/iwrf:data-matthew-input-obs.docker
```

Comment:
```suggestion
    docker create --name ${OBS_DATA_VOL} ncar/iwrf:${OBS_DATA_VOL}.docker
```
I recommend using the env var value to get the obs data so it can be more easily adapted for other use cases.